documentation/kyverno-cli.md

```diff
@@ -74,5 +75,14 @@ Apply multiple policies to multiple resources:
 kyverno apply /path/to/policy1.yaml /path/to/folderFullOfPolicies --resource /path/to/resource1.yaml --resource /path/to/resource2.yaml --cluster
 ```
 
+Saving the mutated resource in a file:
+```
+kyverno apply /path/to/policy.yaml --resource /path/to/resource.yaml -f <file path>
```

Comment:
Can't we have a single flag and then internally check if its a file or dir? Like kubectl?

Reply:
updated accordingly